### ISSUE TITLE
Two apps with Firestore running at the same time crash Unity because of DB Lock

### ISSUE THREAD

**Issue by SummerLynx** [firebase/quickstart-unity#934]
### [REQUIRED] Please fill in the following fields:

  * Unity editor version: 2020.2.0b14.3668
  * Firebase Unity SDK version: 7.0.1
  * Source you installed the SDK: .unitypackage 
  * Problematic Firebase Component: Firestore 
  * Other Firebase Components in use: App, Auth, Analytics, Firestore, Storage
  * Additional SDKs you are using: None
  * Platform you are using the Unity editor on: Windows
  * Platform you are targeting: Editor (Android and desktop)
  * Scripting Runtime: (Editor) IL2CPP

### [REQUIRED] Please describe the issue here:

I have 2 Unity Projects with 2 Firebase Apps: Android Client and Desktop Windows "Server". Both use Firestore DB. 
If i try to run them simultaneously, the one that runs later will crash because the first one locks the Firebase Project Database files.

#### Steps to reproduce:

1. Create two projects.
2. Install Firestore to both of them.
3. Run both of them in parallel.
4. Observe a crash

100% repro rate? 

Unity editor log:

ERROR: FIRESTORE INTERNAL ASSERTION FAILED: z:\build\work\80921a29dd0738241effef99016713332897\google3\third_party\firebase\ios\Releases\FirebaseFirestore\core\src\core\firestore_client.cc(187) void __cdecl firebase::firestore::core::FirestoreClient::Initialize(const class firebase::firestore::auth::User &,const class firebase::firestore::api::Settings &): Failed to open DB: Internal: Failed to open LevelDB database at C:\Users\USER\AppData\Local\firestore\__FIRAPP_DEFAULT\firebase-project-name\main: LevelDB error: IO error: C:\Users\USER\AppData\Local\firestore\__FIRAPP_DEFAULT\firebase-project-name\main/LOCK: Process can not get access to the file, because it's locked by another process

 (expected created.ok())

**Comment by SummerLynx** [comment]
Temporary workaround is to disable Persistence with:

FirebaseFirestore.DefaultInstance.Settings.PersistenceEnabled = false;

**Comment by chkuang-g** [comment]
Duplicate of #928

I think this is the same issue to #928. I'll close this issue as duplication. Please follow the other thread for updates.
